website/docs/r/s3_bucket.html.markdown

```diff
@@ -450,6 +450,7 @@ The following attributes are exported:
 * `id` - The name of the bucket.
 * `arn` - The ARN of the bucket. Will be of format `arn:aws:s3:::bucketname`.
 * `bucket_domain_name` - The bucket domain name. Will be of format `bucketname.s3.amazonaws.com`.
+* `bucket_regional_domain_name` - The bucket region-specific domain name. Will be of format `bucketname.s3-eu-west-1.amazonaws.com`. Note: The AWS CloudFront allows specifying S3 region-specific endpoint when creating S3 origin, it will prevent [redirect issues](https://forums.aws.amazon.com/thread.jspa?threadID=216814) from CloudFront to S3 Origin URL.
```

Comment:
Rather than specifying `bucketname.s3-eu-west-1.amazonaws.com` here we should probably instead point to the [AWS regions and endpoints documentation](https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region) mentioning the domain name including the region name.

Reply:
👍